vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderGroup.kt

```diff
@@ -251,27 +251,6 @@ class RoomListSectionBuilderGroup(
                            @StringRes nameRes: Int,
                            notifyOfLocalEcho: Boolean = false,
                            query: (RoomSummaryQueryParams.Builder) -> Unit) {
-<<<<<<< HEAD
-        withQueryParams(
-                { query.invoke(it) },
-                { roomQueryParams ->
-                    val name = stringProvider.getString(nameRes)
-                    session.getFilteredPagedRoomSummariesLive(roomQueryParams)
-                            .also {
-                                onUpdatable(it)
```

Comment:
This is a specific tchap change introduced in #107, she should probably restore it (and add a tchap comment) or check if it is still necessary